pkg/api/v1/conversion.go

```diff
@@ -405,7 +417,6 @@ func Convert_api_Pod_To_v1_Pod(in *api.Pod, out *Pod, s conversion.Scope) error
 	if err := autoConvert_api_Pod_To_v1_Pod(in, out, s); err != nil {
 		return err
 	}
-
```

Comment:
must have been when I was moving the conversions around because annotations were getting dropped.  Will revert